homeassistant/components/pioneer/media_player.py

```diff
@@ -2,9 +2,7 @@
 import logging
 import telnetlib
```

Comment:
Before we go ahead with this PR this integration needs to use a 3rd party library specific for pioneer devices. The library should handle the device specific interface code and data parsing.

https://developers.home-assistant.io/docs/creating_component_code_review#4-communication-with-devicesservices